src/Microsoft.SqlTools.ServiceLayer/Utility/CommandOptions.cs

```diff
@@ -26,18 +28,31 @@ public CommandOptions(string[] args)
                     string arg = args[i];
                     if (arg.StartsWith("--") || arg.StartsWith("-"))
                     {
+                        // Extracting arguments and properties
                         arg = arg.Substring(1).ToLowerInvariant();
-                        switch (arg)
+                        string argName = arg;
+                        string argProperty = "";
+                        int splitIndex = arg.IndexOf(' ');
```

Comment:
The convention for dotnet command line arguments is `--myarg property`. 
In this case I am using `--locale enu`. If the argument has a space to split by, then I split it into its argument name and property, otherwise i just keep it as an argument with an empty property. We were always just using arguments with no properties. so this wasn't needed until now.